cli_tools_tests/module/diskinspect/disk_inspect_test.go

```diff
@@ -228,7 +228,7 @@ func TestInspectDisk(t *testing.T) {
 			imageURI: "projects/compute-image-tools-test/global/images/windows-2019",
 			expected: &pb.InspectionResults{
 				OsRelease: &pb.OsRelease{
-					CliFormatted: "windows-2019-x64",
```

Comment:
Was there a bug earlier (regarding the extra "-x64") that was not detected?

Reply:
The tests were stale, and I didn't see them until running them on this PR. (I'm still trying to figure out automation for the tests in `cli_tools_tests` directory. I created b/176264612 to fix this.)